index.html

```diff
@@ -63,5 +63,175 @@ <h6><a href="#">LET’S CONNECT</a></h6>
     </section>
   </div>
 
+  <section id="work">
+    <div class="work-content">
+      <div class="card">
+        <img src="images/mobile/work/portfolio-1.png" alt="project screenshot"/>
+        <div class="card-text">
+          <div class="text">
+            <h3>Tonic</h3>
+            <div class="frame-2">
+              <ul>
+                <li>CANOPY</li>
+                <li class="role"><img src="images/mobile/work/Counter.png" alt="image" class="counter-1"/><h6>Back End Dev</h6></li>
+                <li class="year"><img src="images/mobile/work/Counter.png" alt="image" class="counter-2"/><h6>2015</h6></li>
+              </ul>
+            </div>
+          </div>
+
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul>
```

Comment:
As highlighted in [the previous review](https://github.com/Emmy-github-webdev/portfolio/pull/2#discussion_r711414061), you have not used the correct background color 

- [ ] Please change the background color as defined in the templated.
Your background-color is 
![ogah-bk-new](https://user-images.githubusercontent.com/36542986/133886780-d340824e-457a-4cce-948e-8427fe630888.jpg)
But the design template has this background-color: 
![ogah-bk-orig](https://user-images.githubusercontent.com/36542986/133886792-f705c849-53d8-4f0e-8c7e-8349f784a6c5.jpg)